src/routes/user.route.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import { loginUser } from '../controllers/user.login';
+import { registerUser } from '../controllers/user.register'; 
+
+const userRoutes = express.Router();
+
+userRoutes.post('/users/register', registerUser);
```

Comment:
I recommend to prefix all endpoints with `/api/v1/`, you don't have to but its good practice.